src/pre.js

```diff
@@ -229,6 +235,9 @@ var glpkPromise = new Promise(function (resolve) {
 					for (i = 1, ii = glp_get_num_cols(P); i < ii + 1; i++) {
 						res.vars[glp_get_col_name(P, i)] = glp_get_col_prim(P, i);
 					}
+					for (i = 1, ii = glp_get_num_rows(P); i < ii + 1; i++) {
+						res.cons[glp_get_row_name(P, i)] = {'dual': glp_get_row_dual(P, i)};
```

Comment:
also, if we are adding this, we also need to update the types in https://github.com/TakeScoop/glpk.js/blob/master/glpk.d.ts

Reply:
I believe you do not need the `'`